modular/gater/auth_handler.go

```diff
@@ -104,24 +105,23 @@ func (g *GateModular) updateUserPublicKeyHandler(w http.ResponseWriter, r *http.
 		}
 	}()
 
-	reqCtx, err = NewRequestContext(r, g)
-	if err != nil {
-		// verify personal sign signature
-		personalSignSignaturePrefix := signaturePrefix(SignTypePersonal, SignAlgorithm)
-		requestSignature := reqCtx.request.Header.Get(GnfdAuthorizationHeader)
-
-		if strings.HasPrefix(requestSignature, personalSignSignaturePrefix) {
-			accAddress, personalSignVerifyErr := verifyPersonalSignatureFromHeader(requestSignature[len(personalSignSignaturePrefix):])
-			if personalSignVerifyErr != nil {
-				log.CtxErrorw(reqCtx.Context(), "failed to verify signature", "error", personalSignVerifyErr)
-				err = personalSignVerifyErr
-				return
-			}
-			account = accAddress.String()
-			reqCtx.account = account
-		} else {
+	reqCtx, _ = NewRequestContext(r, g)
+	// verify personal sign signature
+	personalSignSignaturePrefix := signaturePrefix(SignTypePersonal, SignAlgorithm)
+	requestSignature := reqCtx.request.Header.Get(GnfdAuthorizationHeader)
+
+	if strings.HasPrefix(requestSignature, personalSignSignaturePrefix) {
+		accAddress, personalSignVerifyErr := verifyPersonalSignatureFromHeader(requestSignature[len(personalSignSignaturePrefix):])
```

Comment:
`verifyPersonalSignatureFromHeader` this is a very strange function name, as its logic does not related to any header, but its name ended with `FromHeader`.  When we name a function, where the parameter comes from dont have to be part of its name.

Reply:
Done